pkg/orchestration/wiring/wiringutil/iam/iam.go

```diff
@@ -58,12 +58,11 @@ func PluginServiceInstance(computeType iam_plugin.ComputeType, resourceName voya
 				},
 			},
 		},
-		Exposed: false, // Why anything would want to depend on ServiceInstance that provisions the IAM role?
 	}
 
 	return instanceResource, nil
 }
 
 func ServiceBinding(compute voyager.ResourceName, iamPluginServiceInstance smith_v1.ResourceName) wiringplugin.WiredSmithResource {
-	return wiringutil.ResourceInternalServiceBinding(compute, iamPluginServiceInstance, namePostfix, true)
```

Comment:
@ash2k can this be a problem? we no longer expose this binding

Reply:
What do you mean? We no longer provide a way for consumer to depend on something directly.